app/views/users/show.html.erb

```diff
@@ -6,7 +6,18 @@
   </head>
```

Comment:
- [ ] Please make sure that the post index page matches the design, same as before the posts should be in boxes as in the design the button should have a shadow and a white background, you have to add an image placeholder beside the user info card also please remove this title which is not in the design at all.
![image](https://user-images.githubusercontent.com/74744787/216771176-f2704de6-848f-4e13-8e21-5114ec63f26f.png)
![image](https://user-images.githubusercontent.com/74744787/216771373-1d471d72-b564-4867-a2c0-9eaef4f11240.png)

![image](https://user-images.githubusercontent.com/74744787/216771189-e58efa45-1225-4a56-a6ca-1bd044d7e99f.png)